manifests/server/config.pp

```diff
@@ -147,6 +147,17 @@
                   Exec['puppet_server_config-create_ssl_dir'],
                   ],
     }
+  } elsif $::puppet::server::ca_crl_sync {
+    # If not a ca AND sync the crl from the ca master
+    if $::server_facts['servername'] != undef {
```

Comment:
Sorry, I forgot $::puppetmaster is from the ENC. There's also $::servername which is provided by the master.